### PR TITLE
Fixes #708 Use ButterKnife BindViews in other activities

### LINKED ISSUE #708: Usage of ButterKnife annotation throughout the project
**Actual Behaviour**

Though ButterKnife library dependencies are included and used in some parts of the code, there are other jave classes where the same can be done.

**Expected Behaviour**

Binding of views through out the project to make the project readable, uniform and easy to debug.

**Steps to reproduce it**

Not applicable

**LogCat for the issue**

Not applicable

**Screenshots of the issue**

Not applicable

**Would you like to work on the issue?**

YES.

### PR DESCRIPTION
**Pull request by Rimjhim28**
Fixes issue #708 
Used ButterKnife library in the remaining actvities to maintain uniformity in the code.

Changes:
Included BindView annnotations for the initializarions of views.

Screenshots for the change: 
Not applicable

### CODE REVIEW

**Comment by Rimjhim28** [comment]
@CloudyPadmal @asitava1998 Please review the changes. Let me know if further changes are required.

**Comment by Rimjhim28** [comment]
@asitava1998 I have tried to solve the errors as much as I could but there are some that still exist. Can you please help in resolving those.

**Comment by asitava1998** [comment]
@Rimjhim28  The code you wrote is slightly incomplete. You missed an essential step. You also need to import `butterknife.ButterKnife` and mention `ButterKnife.bind(this);` in the `OnCreate` method of each file. 
The file `LogicalAnalyzerActivity.java` has an example of butterknife implemented properly. Please see that file and make the necessary changes.

**Comment by CloudyPadmal** [comment]
Hi! Can you provide an APK to test the changes?

**Comment by Rimjhim28** [comment]
Hey @CloudyPadmal! Here's it:
[app-debug.zip](https://github.com/fossasia/pslab-android/files/1677722/app-debug.zip)

File: app/src/main/java/org/fossasia/pslab/activity/PerformExperimentActivity.java [hunk 1]
```diff
@@ -12,28 +12,34 @@
 import org.fossasia.pslab.R;
 import org.fossasia.pslab.adapters.PerformExperimentAdapter;
 
+import butterknife.BindView;
+
```
Comment (asitava1998) [inline app/src/main/java/org/fossasia/pslab/activity/PerformExperimentActivity.java]: You missed the `butterknife.Butterknife` import here and also the bind command. As a result, when an experiment is opened in the apk you provided, the app crashes.

**Comment by Rimjhim28** [comment]
@asitava1998 @CloudyPadmal 
Updated the PR. Here's the new apk:
[app-debug.zip](https://github.com/fossasia/pslab-android/files/1677786/app-debug.zip)

**Pull request merged**